readme.md

```diff
@@ -81,7 +82,8 @@ Your Rails database has two tables. `students` has the columns `id` and `name`,
 Use ActiveRecord to create a new `pbj` sandwich and make it belong to the student named Geraldo.
 
 ```rb
-# Your answer...
+pbj = pbj.new
```

Comment:
[Take a look at the solution.](https://github.com/ga-dc/wdi7_quiz10/tree/solution#question-6)